src/pages/Compendia/DownloadCompendia.js

```diff
@@ -33,6 +34,27 @@ let DownloadCompendia = ({
     })
   );
 
+  const trackDownloadType = compendium => {
+    if (process.env.NODE_ENV === 'production') {
+      const type = compendium.quant_sf_only ? 'RNA-seq' : 'Normalized';
+      ReactGA.event({
+        category: 'Compendia',
+        action: `${type} Download`,
+        label: compendium.primary_organism,
```

Comment:
Can we also track the compendium versions and the API token that was used to retrieve it?

Reply:
added tracking creation of api tokens